DiamondLightSource/hyperion
# Get feature flags from config service

Fixes #1397 

Does a best effort attempt at getting feature flags (`set_stub_offsets`) from the new config service

### To test:
1. Have a look at the config service repo and play with the example
2. Run the new system tests
3. See that nothing else is broken
